docs/_articles/en/user-guide/org-browser.md

```diff
@@ -19,7 +19,7 @@ The Org Browser displays the available metadata types and their corresponding co
 8. Retrieve source for all components of metadata type
 9. Retrieve source for metadata component and open it in VS Code
 
-To open the Org Browser, click the cloud icon in the side bar of the VS Code window **(1)**. If you don't see the icon, make sure your [default org](./en/user-guide/default-org/) is set to a non-scratch org.
+To open the Org Browser, click the cloud icon in the sidebar of the VS Code window **(1)**. If you don't see the icon, make sure your [default org](./en/user-guide/default-org/) is set to a non-scratch org.
```

Comment:
@smaddox-sf @AnanyaJha  - what about sandboxes with source tracking enabled? does that work? Wondering if it's the fact that it's a scratch org - or if it's source tracking that isn't copacetic with Org Browser.

Reply:
Oh boy, @sbudhirajadoc - Pandora's box alert. Looking at VS Code docs, I suspected that they spelled "side bar" as two words - and that's why our docs have used that convention. But looking at their UI topic, that left panel is called the Activity Bar. The bottom of the screen is the Side Bar (but seriously, shouldn't it be bottom bar, heh) I do think we should follow suit to avoid confusion. https://code.visualstudio.com/docs/getstarted/userinterface

Reply:
Apologies for not noticing this sooner.

Reply:
> @smaddox-sf @AnanyaJha - what about sandboxes with source tracking enabled? does that work? Wondering if it's the fact that it's a scratch org - or if it's source tracking that isn't copacetic with Org Browser.

On it. I'll fix this. This text should have been updated already. I have covered this with Stephanie in a recent Code Builder related discussion only recently.